.changeset/brave-colts-cover.md

```diff
@@ -0,0 +1,25 @@
+---
+"astro": minor
+---
+
+It's now possible the render components in isolation with the new Astro Container APIs:
+
+If you use `vitest`, you can quickly write a test like this:
```

Comment:
```suggestion
Introduces an experimental Container API to render `.astro` components in isolation.

This API introduces three new functions to allow you to create a new container and render an Astro component returning either a string or a Response:

- `create()`: creates a new instance of the container.
- `renderToString()`: renders a component and return a string.
- `renderToResponse()`: renders a component and returns the `Response` emitted by the rendering phase.

The first supported use of this new API is to enable unit testing. For example, with  `vitest`, you can create a container to render your component with test data and check the result:
```
OK, make all this correct and make sense, but I'm thinking maybe something like this here? Something approx this length and structure would work, I think?